src/apis/doctor/doctor.service.js

```diff
@@ -368,7 +359,23 @@ class DoctorService {
         });
     }
 
-    /* for doctor activity */
+    updateDoctorStoreByDoctorId(id, type1, value1, callback) {
+            doctorStoreModel.doctor_store.findAll({ where: { userId: id, type: type1 } }).then((doctorstore) => {
+                if (doctorstore.length === 0) {
+                    let doctorStore = { userId: id, type: type1, value: value1 }
+                    this.createDoctorStore(doctorStore, (e) => { log.info("Doctor store created for Doctor id " + id + " for" + type1) })
+                } else {
+                    return doctorStoreModel.doctor_store.update({ value: value1 }, { where: { userId: id, type: type1 } }, (doctorStoreDeleted) => {
+                        callback(doctorStoreDeleted);
```

Comment:
Are you deleting any entry here or is it just the name? If its just the name, kindly change it.

Reply:
No it is just updating it

Reply:
Make the respective changes in that case.